icu4c/source/test/intltest/numbertest_simple.cpp

```diff
@@ -0,0 +1,46 @@
+// © 2018 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#include "unicode/utypes.h"
+
+#if !UCONFIG_NO_FORMATTING
+
+#include "numbertest.h"
+#include "unicode/numberformatter.h"
+
+#include <cmath>
+#include <numparse_affixes.h>
+
+
+void SimpleNumberFormatterTest::runIndexedTest(int32_t index, UBool exec, const char*& name, char*) {
```

Comment:
Remember to hook it into itformat.cpp or similar, so that it actually gets called.

Reply:
It is, in numbertest.h